splinepy/helpme/reparameterize.py

```diff
@@ -3,7 +3,7 @@
 from splinepy import utils as _utils
 
```

Comment:
It's about the filename: `reparametrize.py`

Reply:
deleted the `e`, hope that's what you meant